src/commands/cronjob/update.tsx

```diff
@@ -86,28 +81,7 @@ export default class Update extends ExecRenderBaseCommand<
       disable,
     } = this.flags;
 
-    if (timeout) {
-      checkTimeout(timeout);
-    }
```

Comment:
I removed the check. Im making an effort to get a more helpful error message from the api or be able to read limits from the spec and use them in the client to be able to use checks that sync themselves.